source/steamshim/src/os.cpp

```diff
@@ -92,20 +111,27 @@ int CALLBACK WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance,
 #endif
 
 #else  // everyone else that isn't Windows.
+#include <unistd.h>
 
 bool launchChild(ProcessType *pid, const char* name )
 {
+
+    char exename[32] = "./";
+    strncpy(exename+2, name, 30);
+
+
+    if (access(exename, F_OK) < 0) return false;
```

Comment:
Both this one (and the windows one) depend on the working directory being set correctly.
Is that an assumption that can be made here?

Reply:
yes, the game will already implode if not